mkosi

```diff
@@ -427,16 +434,21 @@ def install_fedora(args, workspace, run_build_script):
     if os.path.exists(gpg_key):
         gpg_key = "file://%s" % gpg_key
     else:
-        gpg_key = "https://getfedora.org/static/81B46521.txt"
+        gpg_key = "https://getfedora.org/static/{key}.txt".format(key=FEDORA_KEYS_MAP[args.release])
 
     if args.mirror:
-        release_url = "baseurl={.mirror}/releases/{.release}/Everything/x86_64/os/".format(args)
-        updates_url = "baseurl={.mirror}/updates/{.release}/x86_64/".format(args)
+        if args.release == "25":
+            section = "development"
+        else:
+            section = "releases"
+
+        release_url = "baseurl={args.mirror}/{section}/{args.release}/Everything/x86_64/os/".format(args=args, section=section)
+        updates_url = "baseurl={args.mirror}/updates/{args.release}/x86_64/".format(args=args)
```

Comment:
That's probably better than what was before..., but it feels suboptimal. By the time this mkosi is released, Fedora will likely have released F25, and this will be broken. Maybe we should do a query of the URL (using `requests` or `urllib`), and pick the URL that works?